parachains/runtimes/collectives/collectives-polkadot/src/fellowship/mod.rs

```diff
@@ -106,12 +106,15 @@ impl pallet_ranked_collective::Config<FellowshipCollectiveInstance> for Runtime
 	type WeightInfo = weights::pallet_ranked_collective::WeightInfo<Runtime>;
 	type RuntimeEvent = RuntimeEvent;
 	// Promotions and the induction of new members are serviced by `FellowshipCore` pallet instance.
-	type PromoteOrigin = EnsureRootWithSuccess<Self::AccountId, ConstU16<{ ranks::DAN_9 }>>;
+	// The maximum value of `u16` set as a success value for the root to ensure the benchmarks will pass.
+	type PromoteOrigin = EnsureRootWithSuccess<Self::AccountId, ConstU16<65535>>;
```

Comment:
should we have this behind `#[cfg(feature = "runtime-benchmarks")]` and have DAN_9 if the feature is not enabled?

Reply:
If this was pulled out into a type def then that type could also be referenced in the DemoteOrigin

Reply:
I liked this way for simplicity, both ways looks ok to me.

Reply:
i added a separate promote origin for benches, please approve if looks good